api/controller.py

```diff
@@ -271,7 +271,9 @@ def authenticated_patron_from_request(self):
         try:
             patron = self.authenticated_patron(header.username, header.password)
         except RemoteInitiatedServerError,e:
-            return problem(REMOTE_INTEGRATION_FAILED, e.message, 503)
```

Comment:
REMOTE_INTEGRATION_FAILED is already a ProblemDetail. This was creating a ProblemDetail that contained another ProblemDetail, which makes no sense.